src/features/defi/helpers/utils.ts

```diff
@@ -12,3 +16,17 @@ export const chainIdToLabel = (chainId: ChainId): string => {
     }
   }
 }
+
+export const makeHasEnoughBalanceForGas = (
+  feeAsset: Asset,
+  estimatedGasCrypto?: string | undefined,
+) => {
+  const state = store.getState()
+  const feeAssetBalance = selectPortfolioCryptoHumanBalanceByAssetId(state, {
+    assetId: feeAsset?.assetId ?? '',
+  })
```

Comment:
`feeAssetBalance` is not passed directly, so we can leverage passing only `feeAsset` and deriving the balance from it